keanu-project/src/main/java/io/improbable/keanu/model/regression/RegressionModelBuilder.java

```diff
@@ -145,14 +162,6 @@ private DoubleVertex getWeightsVertex() {
         return this.regularization.getWeightsVertex(getFeatureCount(), priorOnWeightsMeans, priorOnWeightsScaleParameters);
     }
 
-    private void performDataFitting(LinearRegressionGraph<OUTPUT> regressionGraph, OUTPUT outputTrainingData) {
-        ModelFitter<DoubleTensor, OUTPUT> fitter = samplingAlgorithm == null ?
-            this.regularization.createFitterForGraph(regressionGraph) :
-            samplingAlgorithm.createFitterForGraph(regressionGraph);
-
-        fitter.fit(inputTrainingData, outputTrainingData);
```

Comment:
If you no longer do the fitting in the builder, you will need to update `SimpleLinearRegressionExample`